### ISSUE TITLE
Add "use of closed network connection" as reason to retry

### ISSUE THREAD

**Issue by rleighton** [panther-labs/panther#1453]
### Description

We got this error: 
```
failed to send notification to topic: RequestError: send request failed
caused by: Post "https://sns.us-west-2.amazonaws.com/": write tcp 169.254.76.1:59092->52.119.170.155:443: use of closed network connection
```

This should be added to the retry logic for the log processor

**Comment by kostaspap** [comment]
Seems there is a Github issue on AWS SDK related to this: https://github.com/aws/aws-sdk-go/issues/3406

**Comment by kostaspap** [comment]
Ok, so it seems that AWS fixed it in version 1.34.3. We upgraded to that version of the AWS SDK here: https://github.com/panther-labs/panther/commit/bfa70304612688f7c9f2213619a6f8fab4dbd98a 
This means that the fix will be part of the 1.8 release. No further actions required.
